xmppserver/src/main/webapp/connection-settings-external-components.jsp

```diff
@@ -440,30 +438,30 @@
 <fmt:message key="component.settings.disallowed" var="disallowedTitle"/>
 <admin:contentBox title="${disallowedTitle}">
     <p><fmt:message key="component.settings.disallowed.info" /></p>
-    <table class="jive-table" cellpadding="3" cellspacing="0" border="0" >
+    <table class="jive-table" >
         <tr>
-            <th width="1%">&nbsp;</th>
-            <th width="89%" nowrap><fmt:message key="component.settings.subdomain" /></th>
-            <th width="10%" nowrap><fmt:message key="global.delete" /></th>
+            <th style="width: 1%">&nbsp;</th>
+            <th style="width: 89%; white-space: nowrap"><fmt:message key="component.settings.subdomain" /></th>
+            <th style="width: 10%; white-space: nowrap"><fmt:message key="global.delete" /></th>
         </tr>
         <c:choose>
             <c:when test="${empty blockedComponents}">
                 <tr>
-                    <td align="center" colspan="7"><fmt:message key="component.settings.empty_list" /></td>
+                    <td style="text-align: center" colspan="7"><fmt:message key="component.settings.empty_list" /></td>
                 </tr>
             </c:when>
             <c:otherwise>
                 <c:forEach var="component" varStatus="status" items="${blockedComponents}">
-                    <tr class="${ ( (status.index + 1) % 2 ) eq 0 ? 'jive-even' : 'jive-odd'}">
+                    <tr>
                         <td>${ status.index + 1}</td>
                         <td><c:out value="${component.subdomain}"/></td>
-                        <td align="center" style="border-right:1px #ccc solid;">
+                        <td style="border-right:1px #ccc solid; text-align: center">
```

Comment:
This delete icon is squiff

![image](https://user-images.githubusercontent.com/2117083/203164466-8c08c38c-1c86-49a6-aa7a-00e1cbf4c2ba.png)

Reply:
The headers needs to have a similar text-align as the data.